src/Valit/Strategies/ValitStrategies.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Runtime.CompilerServices;
+
+namespace Valit.Strategies
+{
```

Comment:
Can you explain what's happening here? I'm not quite sure :/

Reply:
I need an object with two properties here for a "picker" syntax of `Func<DefaultValitStrategies, IValitStrategy>.`
so we could have `selections => selections.MySelection`

So I have couple of options for initializing this... 
1. I could have made a singleton there like DefaultValitStrategies.Once, make these strategies a properties of it and  passed a reference to it into `Func<>`
2. Or do what I did and have weightless (in bytesize per instance) struct with properties accessing static members. There is no point of **not** using inline here, as we are accessing static readonly "intentionally" immutable members.
resulting syntax for the user looks the same as I would pass a reference to a class with the difference being, it is not a reference type which is a tiny win in this case.